### ISSUE TITLE
Unload don't create files while concurrency scaling enabled

### ISSUE THREAD

**Issue by maxim-lisovsky-gismart** [capitalone/locopy#231]
When we enabled concurrency scaling in redshift `locopy.redshift.unload_and_copy` stopped creating files on S3 during critical load and logged the corresponding message:

> No files generated from unload

Tried both `parquet` and `csv` formats

**Comment by fdosani** [comment]
Could you provide more details? Like a minimal example of the code and stack trace etc. it's hard to debug just based on what you've provided. TIA!

**Comment by fdosani** [comment]
Also do any of these limitations apply to you setup? https://docs.aws.amazon.com/redshift/latest/dg/concurrency-scaling.html

**Comment by maxim-lisovsky-gismart** [comment]
I do something like this:
```
query = "select * from foo.bar"
with connection.get_redshift(secret_id, database=database, host=host) as redshift_locopy:
    redshift_locopy.unload_and_copy(
        query=query,
        s3_bucket=bucket,
        s3_folder=bucket_dir,
        export_path=False,
        raw_unload_path=data_dir,
        delete_s3_after=False,
        parallel_off=False,
        unload_options=["PARQUET", "PARALLEL ON", "CLEANPATH"],
    )
```
Haven't found any of these restrictions relevant for me

**Comment by fdosani** [comment]
I'm not sure this is a `locopy` issue though. Sounds like concurrency scaling is changing the behaviour of redshift which means the data it is querying isn't available to download? I think one of those limitations is applying to your setup. It works without it enabled correct?

**Comment by maxim-lisovsky-gismart** [comment]
Yes, it works without concurrency. I also have an idea that it can be caused by this:
```
SELECT path FROM stl_unload_log WHERE query = pg_last_query_id() ORDER BY path;
```
Probably last query id is not so reliable when it comes to concurrency

**Comment by fdosani** [comment]
It say this in the docs: 

> It doesn't support queries that access system tables, PostgreSQL catalog tables, or no-backup tables.

That would make sense.

**Comment by maxim-lisovsky-gismart** [comment]
But it's locopy's code

**Comment by fdosani** [comment]
> But it's locopy's code, `unload_and_copy` method

If you don't use `locopy` and have concurrency scaling enabled does your query work.
